zebracrossing/templates/zebracrossing/base.html

```diff
@@ -10,10 +10,16 @@
     {% bootstrap_css %}
     <link href="{% static 'css/zebracrossing.css' %}" rel="stylesheet" media="screen">
     <link rel="stylesheet" href="{% static 'fontawesomefree/css/all.min.css' %}">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
```

Comment:
This should fetch the local copy you added as a dependency and be in the campaign view only